charts/digibyte/values.yaml

```diff
@@ -4,7 +4,7 @@ replicaCount: 1
 image:
   repository: arrajput/digibyte
   pullPolicy: Always
-  tag: "2.5"
+  tag: "latest"
```

Comment:
Charts should always use a versioned image, let's update to the right tag instead of latest.

Reply:
Done as requested